projects/1.19/assets/sweetmagic/sweetmagic/lang/zh_cn.json

```
@@ -0,0 +1,1097 @@
+	"item.sweetmagic.flour": "面粉",
+	"item.sweetmagic.soybean_flour": "黄豆粉",
+	"item.sweetmagic.butter": "黄油",
+	"item.sweetmagic.custard": "卡仕达酱",
+	"item.sweetmagic.olive_oil": "橄榄油",
+	"item.sweetmagic.whipping_cream": "奶油",
+	"item.sweetmagic.cocoa_powder": "可可粉",
+	"item.sweetmagic.sponge_cake": "海绵蛋糕",
+	"item.sweetmagic.boiled_edamame": "熟毛豆",
+	"item.sweetmagic.sunny_side_up": "煎蛋",
+	"item.sweetmagic.applepie": "苹果派",
+	"item.sweetmagic.toast": "吐司面包",
+	"item.sweetmagic.butter_toast": "黄油吐司",
+	"item.sweetmagic.butter_role": "黄油餐包",
+	"item.sweetmagic.cream_filled_roll": "奶油面包",
+	"item.sweetmagic.croissant": "牛角面包",
+	"item.sweetmagic.melon_bread": "菠萝包",
+	"item.sweetmagic.applecandy": "苹果糖",
+	"item.sweetmagic.pretzel": "蝴蝶脆饼",
+	"item.sweetmagic.cup_cake": "纸杯蛋糕",
+	"item.sweetmagic.french_toast": "法式吐司",
+	"item.sweetmagic.fried_potato": "炸薯条",
+	"item.sweetmagic.icing_cookies": "糖霜曲奇",
+	"item.sweetmagic.jam_toast": "果酱吐司",
+	"item.sweetmagic.kinakomochi": "黄豆粉麻薯",
+	"item.sweetmagic.mochi": "麻薯",
+	"item.sweetmagic.potatobutter": "黄油土豆",
+	"item.sweetmagic.pumpkin_nituke": "煮南瓜",
+	"item.sweetmagic.cooked_rice": "米饭",
+	"item.sweetmagic.omelet_rice": "蛋包饭",
+	"item.sweetmagic.scone": "司康饼",
+	"item.sweetmagic.strawberry_tart": "草莓挞",
+	"item.sweetmagic.omelet": "玉子烧",
+	"item.sweetmagic.waffle": "华夫饼",
+	"item.sweetmagic.baked_sweetpotato": "烤红薯",
+	"item.sweetmagic.baked_eggplant": "烤茄子",
+	"item.sweetmagic.zunda": "毛豆泥麻薯",
+	"item.sweetmagic.cheese": "芝士",
+	"item.sweetmagic.yogurt": "酸奶",
+	"item.sweetmagic.baked_banana": "烤香蕉",
+	"item.sweetmagic.baked_corn": "烤玉米",
+	"item.sweetmagic.orange_jelly": "柑橘果冻",
+	"item.sweetmagic.strawberry_jelly": "草莓果冻",
+	"item.sweetmagic.apple_jelly": "苹果冻",
+	"item.sweetmagic.peach_jelly": "桃子果冻",
+	"item.sweetmagic.lemon_shaved_ice": "柠檬刨冰",
+	"item.sweetmagic.strawberry_shaved_ice": "草莓刨冰",
+	"item.sweetmagic.condensed_milk_shaved_ice": "炼乳刨冰",
+	"item.sweetmagic.cake_roll": "蛋糕卷",
+	"item.sweetmagic.short_cake": "草莓松饼",
+	"item.sweetmagic.salad_caprese": "意式番茄沙拉",
+	"item.sweetmagic.sandwitch_egg": "鸡蛋三明治",
+	"item.sweetmagic.corn_soup": "玉米浓汤",
+	"item.sweetmagic.strawberry_milk": "草莓牛奶",
+	"item.sweetmagic.coconut_juice": "椰子汁",
+	"item.sweetmagic.pumpkin_soup": "南瓜浓汤",
+	"item.sweetmagic.banana_smoothy": "香蕉奶昔",
+	"item.sweetmagic.mixed_juice": "混合奶昔",
+	"item.sweetmagic.coffee": "咖啡",
+	"item.sweetmagic.cafe_latte": "拿铁咖啡",
+	"item.sweetmagic.vienna_coffee": "维也纳咖啡",
+	"item.sweetmagic.cocoa": "热可可",
+	"item.sweetmagic.alternative_ingot": "亚钢锭",
+	"item.sweetmagic.alternative_axe": "亚钢斧",
+	"item.sweetmagic.alternative_pick": "亚钢镐",
+	"item.sweetmagic.alternative_shovel": "亚钢锹",
+	"item.sweetmagic.alternative_sickle": "亚钢镰刀",
+	"item.sweetmagic.alternative_shears": "亚钢剪",
+	"item.sweetmagic.skullfrost_spawn_egg": "冰霜骷髅刷怪蛋",
+	"item.sweetmagic.blazetempest_spawn_egg": "风暴烈焰人刷怪蛋",
+	"item.sweetmagic.archspider_spawn_egg": "毒牙蛛刷怪蛋",
+	"item.sweetmagic.electriccube_spawn_egg": "电流史莱姆刷怪蛋",
```

```suggestion
	"item.sweetmagic.electriccube_spawn_egg": "电气史莱姆刷怪蛋",
```
```suggestion
	"item.sweetmagic.electriccube_spawn_egg": "电力史莱姆刷怪蛋",
```
可能致敬游戏王字段`エレキ`（电气/电力）
确定是史莱姆吗（原文是cube